Doppler.Sap/Controllers/BillingController.cs

```diff
@@ -42,5 +42,15 @@ public async Task<IActionResult> CreateBillingRequest([FromBody] List<BillingReq
 
             return new OkObjectResult("Successfully");
         }
+
+        [HttpPatch("UpdateBilling")]
```

Comment:
Why did you choose the _patch_ verb?

What is the full URL?

Is this operation idempotent?

Reply:
I choose "Patch" becuase only I update a few properties of the entity. I have understood that if we need update the entire resource the correct is uses "PUT" and if we need update a few properties of the resource use "PATCH". Maybe my vision can be wrong.  Please, let me know if it is wrong.

The url is: https://localhost:44360/Billing/UpdateBilling

Reply:
@andresmoschini, do you prefer in this case use the verb: "POST" ?. Thanks.

Reply:
We have talked about this. @jhoffman-ms will apply some changes.

Reply:
@andresmoschini, regarding previous our call I renamed the endpoint. Now the endpoint is: ```Billing/UpdatePaymentStatus``` and the verb is ```POST```.

Thanks for your feedback. Let me know if you have another comment.